### DISCUSSION TITLE
Welcome to lsp4j Discussions!

### DISCUSSION THREAD

**Discussion by jonahgraham** [eclipse-lsp4j/lsp4j#796] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nixel2007** [comment]
Hi there!

My name is Nikita and I am one of the authors of [BSL Language Server](https://github.com/1c-syntax/bsl-language-server) - LS for 1C:Enterprise platform internal languages. I try to build efficient and quick server that could handle huge 1C code base - millions of LOC in the same project :)

I'm with you sinse 0.4.0 (if I remember it correctly) and really interested into the future of lsp4j.

Also we (1c-syntax community) tried to build something like a skeleton for language server on top of the lsp4j with default implementations for request handling, but this project is unfinished and on pause unfortunately.

**Comment by jonahgraham** [comment]
I'm Jonah, one of the Project Leads of the LSP4J project. I do much of the release engineering so feel free to tag me if you have any questions in that category. See the [endgame](https://github.com/eclipse-lsp4j/lsp4j/labels/endgame) issues for step-by-step lists of upcoming versions and look at [releases](https://github.com/eclipse-lsp4j/lsp4j/releases) and [changelog](https://github.com/eclipse-lsp4j/lsp4j/blob/main/CHANGELOG.md) for details of changes.

[Contributions welcome!](https://github.com/eclipse-lsp4j/lsp4j/blob/main/CONTRIBUTING.md)
